### PR TITLE
Issue#3 - Correction of the error "system exception (code 0x0000008)" 

### PR DESCRIPTION
**Pull request by godindev**
Correction of the error "system exception (code 0x0000008)" when releasing the Response from memory. Sample with dynamic PDF path.

### CODE REVIEW

File: samples/samples.dpr [hunk 1]
```diff
@@ -6,10 +6,19 @@ uses
   Horse,
   Horse.OctetStream,
   System.Classes,
-  System.SysUtils;
+  System.SysUtils,
+  Winapi.Windows;
 
 {$R *.res}
 
+function GetPath: string;
```
Comment (viniciussanchez) [inline samples/samples.dpr]: You cannot use the ExtractFilePath(ParamStr(0)) ?

File: samples/samples.dpr [hunk 2]
```diff
@@ -6,10 +6,19 @@ uses
   Horse,
   Horse.OctetStream,
   System.Classes,
-  System.SysUtils;
+  System.SysUtils,
+  Winapi.Windows;
```
Comment (viniciussanchez) [inline samples/samples.dpr]: ExtractFilePath does not need uses Winapi.Windows

Comment (godindev) [reply samples/samples.dpr]: Yes, sorry I forgot that the code refers to WinApi. I believe that the only case in which ParamStr (0) will not work is ISAPI.

File: samples/samples.dpr (hunk 1 above)
Comment (godindev) [reply samples/samples.dpr]: Yes, in ISAPI ParamStr (0) will not work. If it's not a problem, it's easy to switch.

**Pull request merged**